### PR TITLE
Add support for MNK88 

### PR DESCRIPTION
**Pull request by kopibeng**
## Description

Adds QMK support for MNK88 PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds QMK support for MNK88 PCB

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kopibeng/mnk88/rules.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = STM32F072
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/kopibeng/mnk88/rules.mk:7]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/kopibeng/mnk88/rules.mk (hunk 1 above)
Comment (kopibeng) [reply keyboards/kopibeng/mnk88/rules.mk]: Updated, thanks.

**Review by drashna — APPROVED** [review]

File: keyboards/kopibeng/mnk88/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = STM32F072
```
Comment (fauxpark) [inline keyboards/kopibeng/mnk88/rules.mk:2]:
```suggestion
MCU = STM32F072

# Bootloader selection
```

File: keyboards/kopibeng/mnk88/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# MNK88
+
+![MNK88](https://i.imgur.com/QlKuV3w.png)
```
Comment (fauxpark) [inline keyboards/kopibeng/mnk88/readme.md:3]: Dead link.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kopibeng/mnk88/rules.mk (hunk 2 above)
Comment (kopibeng) [reply keyboards/kopibeng/mnk88/rules.mk:2]: Updated, thanks.

File: keyboards/kopibeng/mnk88/readme.md (hunk 3 above)
Comment (kopibeng) [reply keyboards/kopibeng/mnk88/readme.md]: Updated, thanks.

File: keyboards/kopibeng/mnk88/rules.mk [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+# Bootloader selection
```
Comment (fauxpark) [inline keyboards/kopibeng/mnk88/rules.mk:4]:
```suggestion
# Bootloader selection
BOOTLOADER = stm32-dfu
```
Oops, I was distracted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kopibeng/mnk88/info.json [hunk 5]
```diff
@@ -0,0 +1,120 @@
+{
+    "keyboard_name": "MNK88",
+    "maintainer": "kopibeng",
+    "url": "",
+    "layouts": {
+        "LAYOUT": {
```
Comment (fauxpark) [inline keyboards/kopibeng/mnk88/info.json:6]:
```suggestion
        "LAYOUT_all": {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/kopibeng/mnk88/info.json (hunk 5 above)
Comment (kopibeng) [reply keyboards/kopibeng/mnk88/info.json]: Seems this change is breaking lint check, should it be reverted?

Comment (fauxpark) [reply keyboards/kopibeng/mnk88/info.json:6]: No, you need to fix the lint error.

Comment (kopibeng) [reply keyboards/kopibeng/mnk88/info.json]: Fixed, thanks :)

**Review by fauxpark — APPROVED** [review]

**Pull request merged**